app/models/event_handler.rb

```diff
@@ -1,19 +1,11 @@
 class EventHandler
 
   def receive(event)
-    self.create_event message: "event ##{event.id} was fired", prior_event: event
+    Event.new message: "event ##{event.id} was fired"
```

Comment:
ha, I knew we wouldn't save here!

Reply:
But we weren't going to save in the controller, either.

Things may shift around till they find their proper home.